test/jdk/java/awt/PopupMenu/PopupMenuLocation.java

```diff
@@ -70,11 +74,12 @@ public static void main(final String[] args) throws Exception {
     }
 
     private static void test(final Point tmp) throws Exception {
+        actionEventReceivedLatch = new CountDownLatch(1);
```

Comment:
You set this field on one thread and read on another, some synchronization is needed.

Reply:
Added volatile keyword for this field